### PR TITLE
fix of tuninig of bestfactory

### PR DESCRIPTION
**Pull request by pandaro**

### CODE REVIEW

**Comment by eronoobos** [comment]
thanks for fixing it

btw, i believe you can just add another commit and it will automatically show up in a pull request you currently have open

File: buildsitehandler.lua [hunk 1]
```diff
@@ -416,29 +416,8 @@ function BuildSiteHandler:CheckForDuplicates(unitName)
...
-			local thisIsFactory = unitTable[done.unitName].isBuilding and unitTable[done.unitName].buildOptions
-			if thisIsFactory then
-				if f < done.frame + 600 then
-					-- don't build factories within 20 seconds of one another
-					EchoDebug("found a factory too recently")
-					return true
-				end
-			end
-		end
-	end
```
Comment (eronoobos) [inline buildsitehandler.lua:441]: how is this no longer necessary?

Comment (eronoobos) [inline buildsitehandler.lua:441]: @pandaro i'm still wondering why this was removed

**Comment by pandaro** [comment]
cause a system that simply say me:
to recently factory builded prevent for fast growing factory when we have power do do
if i have already a t2(veh) with lots of metal and energy income, is useless build a t1 bot and wait 50 second to build t3, in this time the t1 veh are freez probably.
better for me is calculate ad hoc the current economic/units situation

**Pull request merged**